build.psm1

```diff
@@ -394,7 +394,8 @@ function Start-BuildNativeUnixBinaries {
     param (
         [switch] $BuildLinuxArm,
         [switch] $BuildLinuxArm64,
-        [switch] $BuildAlpineArm64
```

Comment:
I'm not sure this commit just "Fix typo" 😅 We're adding _both_ AlpineArm64 and AlpineArm now, yeah?

Reply:
lol, yes. Could have been a better commit title.